blocks/story-feed-tag-content-source-block/sources/story-feed-tag.js

```diff
@@ -1,11 +1,39 @@
-/* eslint-disable import/no-unresolved */
-import source from '@arc-core-components/content-source_story-feed_tag-v4';
 import getResizedImageData from '@wpmedia/resizer-image-block';
 
+const params = {
+  tagSlug: 'text',
+  feedSize: 'number',
+  feedOffset: 'number',
+};
+
+/**
+* @func pattern
+* @param {Object} key
+* @return {String} slugified content api search url
+*/
+const pattern = (key = {}) => {
+  const website = key['arc-site'] || 'Arc Site is not defined';
+  const { tagSlug, feedOffset, feedSize } = key;
+
+  if (!tagSlug) {
+    throw new Error('tagSlug parameter is required');
+  }
+
+  return `/content/v4/search/published?q=taxonomy.tags.slug:${tagSlug}&size=${feedSize}&from=${feedOffset}&sort=display_date:desc&website=${website}`;
```

Comment:
`feedSize` and `feedOffset` do not have a default, which causes an error. In my PR I added defaults here to account for this

Reply:
will do thanks

Reply:
updated @matthewroach

Reply:
before 

throws error on tag slug but no feedSize or feedoffset

after 

<img width="1383" alt="Screen Shot 2021-07-26 at 10 51 25" src="https://user-images.githubusercontent.com/5950956/127020308-87ffd72e-104d-4c29-8cee-2dc29556b52e.png">